### DISCUSSION TITLE
Calendar card: Add Weekly view

### DISCUSSION THREAD

**Discussion by bverkron** [home-assistant/frontend#11290] [Other feature requests]
## The request
~~The Weekly and Daily view buttons on the calendar card (top-right corner) are reversed compared to the original calendar in the side panel. The typical Month > Week > Day order would make more sense IMO than the current Month > Day > Week order on the calendar card~~

Edit: Adjusting the request based on comments.

Please add a weekly view option akin to what's available in the original calendar in the side panel, with the days laid out horizontally from left to right.

## The alternatives
None

**Comment by zsarnett** [comment]
The calendar card has a different View than Week View. Its call List View. So its Month > Day > List

**Comment by bverkron** [comment]
Right, noticed that after submitting the issue. Devils advocate it's still scoped to a weekly timeframe so arguably it's still a weekly view.

Counter suggestion would be to add a 4th button for the original horizontal style weekly view and place it between the current monthly and daily buttons. Thoughts?

**Reply by akshaya-a** [reply]
I second this (old) suggestion - I want to cast a calendar horizontal weekly view (actually I want just 3 days, but baby steps..), so I can't use the sidebar Calendar (can I?) and have to use the Calendar card in a lovelace view. Weirdly, the card looks almost identical without the week horizontal view.

**Comment by mpoettgen** [comment]
If I understand this request correctly, then it has been fulfilled in the meantime. The calendar now has month, week, day, and list views.
